controllers/controllers.js

```diff
@@ -0,0 +1,36 @@
+const {
+    fetchTopics,
+    fetchArticles,
+    fetchArticle
+ } = require('../models/models.js')
+
+
+function getTopics(req,res) {
+    fetchTopics()
+        .then((data) => {
+            res.status(200).send(data)
+        })
+}
+
+function getArticles(req,res,next) {
+    if (req.path.endsWith("articles")) {
+        fetchArticles()
+        .then((data) => {
+            res.status(200).send(data)
+        })
+    } else {
+        let index = ((req.path).slice(14))
```

Comment:
This technically works but i want you to look into req.params to get the parametric endpoints and later on, look at req.query to get any queries too